flow-wing-docs/docs/Getting Started/Installation.md

```
@@ -4,6 +4,534 @@ sidebar_position: 1
+The JIT step reuses the AOT-staged `lib/modules/` and `lib/<platform>/` — only the `FlowWing-jit` binary is added.
+
+Shortcut targets that chain configure + build + install:
+
+```bash
+make release-aot RELEASE_INSTALL_PREFIX="/path/to/install"
+make release-jit RELEASE_INSTALL_PREFIX="/path/to/install"
+```
+
+Final install tree:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)